src/Raven.Server/Documents/Sharding/Commands/GetRawTimeSeriesCommand.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using System.IO;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Raven.Client.Documents.Session.Loaders;
+using Raven.Client.Http;
+using Raven.Server.Documents.Sharding.Handlers;
+using Sparrow;
+using Sparrow.Extensions;
+using Sparrow.Json;
+
+namespace Raven.Server.Documents.Sharding.Commands
+{
+    internal class GetRawTimeSeriesCommand : ShardedStreamCommand
```

Comment:
I would prefer to inherit from `GetTimeSeriesCommand` and override the `ProcessResponse`.

To avoid the duplication of the ctor

Reply:
removed this operation